rasa/train.py

```diff
@@ -99,16 +104,26 @@ def train(
 
 
 def train_core(
-    domain: Text, config: Text, stories: Text, output: Text, train_path: Optional[Text]
+    domain: Text,
+    config: Text,
+    stories: Text,
+    output: Text,
+    train_path: Optional[Text],
+    args: Optional[Dict],
 ) -> Optional[Text]:
     loop = asyncio.get_event_loop()
     return loop.run_until_complete(
-        train_core_async(domain, config, stories, output, train_path)
+        train_core_async(domain, config, stories, output, train_path, args)
     )
 
 
 async def train_core_async(
-    domain: Text, config: Text, stories: Text, output: Text, train_path: Optional[Text]
+    domain: Text,
+    config: Text,
+    stories: Text,
+    output: Text,
+    train_path: Optional[Text] = None,
+    args: Optional[Dict] = None,
```

Comment:
why don't we name it `kwargs`?
also on unrelated note, why in `agent.train` it is `**kwargs` while in `rasa.core.train` it is `kwargs`?

Reply:
will rename it

Regarding your question: Is just the way how it is implemented. The `**` operator "unpacks" the keyword arguments. We could also use a `Dict` as in `rasa.core.train`. No major difference.